mitodl/ocw-hugo-themes
# update search

### What are the relevant tickets?
closes https://github.com/mitodl/ocw-hugo-themes/issues/1294

### Description (What does it do?)
This pr updates the search to use the new open apis.

It uses the changes to course-search utils from https://github.com/mitodl/course-search-utils/pull/64

### How can this be tested?
Run mit-open

Set
COURSE_SEARCH_API_URL=http://localhost:8063/api/v1/learning_resources_search/
CONTENT_FILE_SEARCH_API_URL=http://localhost:8063/api/v1/content_file_search/

Run `yarn start www`

Go to http://localhost:3000/search/ and click around. Everything should work as expected. The urls will be a little different than the existing ui
